cmd/minishift/cmd/start.go

```diff
@@ -564,3 +581,168 @@ func IsOpenShiftRunning(driver drivers.Driver) bool {
 
 	return openshift.IsRunning(dockerCommander)
 }
+
+/*
+
+Starting local OpenShift cluster using 'kvm' hypervisor...
+   Memory: 2GB
+   Virtual CPU: 2
+   Diskspace: 20GB persistent storage
+
+*/
+
+/*
+
+Starting locsl OpenShift cluster using '' hypervisor...
+-- Provision hostname
+-- Setting systemd unit files for Docker
+   Restarting Docker ... OK
+-- Mounting hostfolders
+   Mounting 'Disks': //remotemachine/Disks/ ... OK
+-- Provision OpenShift via
+
+*/
+
+/*
+
+ * does the minishift instance have an IP address?
+ * can a remote IP address be reached? (check connectivity)
+ * can a remote host be resolved? (check dns)
+ * is persistent disk mounted
+ * #966 check if driver plugin is available
+
+ */
+
+/*
+
+startupCheck("Checking for ... ", func() bool, shouldFail)
+
+*/
+
+type preflightCheckRunner func() bool
```

Comment:
This will be moved to a new file... and maybe even pkg. Not sure yet... but as part of tests it lives here, as I need to test on three platforms to make sure I get the 'same' results.